pay-api/src/pay_api/resources/invoice_receipt.py

```diff
@@ -57,3 +57,16 @@ def post(payment_id, invoice_id=''):
             return exception.response()
         current_app.logger.debug('>Transaction.post')
         return jsonify(response), 200
+
+    @staticmethod
+    @cors.crossdomain(origin='*')
+    @_jwt.requires_auth
+    def get(payment_id, invoice_id=''):
```

Comment:
Nope, they will not use it. Just used the opportunity to create this endpoint, so that if any new requirements comes we are in position to ask them to use this endpoint.